image/gen/src/generator.rs

```diff
@@ -423,7 +430,7 @@ impl<Crypto: ImageGeneratorCrypto> ImageGenerator<Crypto> {
             r#type: r#type.into(),
             revision: *image.rev(),
             version: image.version(),
-            svn: image.svn(),
+            svn,
```

Comment:
I thought we moved the SVN to a different part of the header. Should the svn field in the individual image entries be deprecated?